BookingsApi/BookingsApi.UnitTests/Controllers/JudiciaryPersonControllerTests.cs

```diff
@@ -0,0 +1,278 @@
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using BookingsApi.Contract.Requests;
+using BookingsApi.Controllers;
+using BookingsApi.DAL.Commands;
+using BookingsApi.DAL.Commands.Core;
+using BookingsApi.DAL.Queries;
+using BookingsApi.DAL.Queries.Core;
+using BookingsApi.Domain;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+using NUnit.Framework;
+
+namespace BookingsApi.UnitTests.Controllers
+{
+    public class JudiciaryPersonControllerTests
+    {
+        protected JudiciaryPersonController Controller;
+        protected Mock<IQueryHandler> QueryHandlerMock;
+        protected Mock<ICommandHandler> CommandHandlerMock;
+        protected Mock<ILogger<JudiciaryPersonController>> LoggerMock;
+
+        [SetUp]
+        public void Setup()
+        {
+            QueryHandlerMock = new Mock<IQueryHandler>();
+            CommandHandlerMock = new Mock<ICommandHandler>();
+            LoggerMock = new Mock<ILogger<JudiciaryPersonController>>();
+            Controller = new JudiciaryPersonController(QueryHandlerMock.Object, CommandHandlerMock.Object, LoggerMock.Object);
```

Comment:
Have you seen `AutoMock`? It's been amazing in creating dependencies automagically without having to manage it yourself

Reply:
Yes used it, don't like all the configuration class and property mappings. I'm sure it works nicely for simple mappings.

Reply:
automock? no configuration or mappings, maybe thinking of automapper?

Reply:
all of the setup can be simplified as
`_mocker = AutoMock.GetLoose();`
`_controller = _mocker.Create<VirtualRoomsController>();`

see the example [here](https://github.com/hmcts/vh-video-web/blob/master/VideoWeb/VideoWeb.UnitTests/Controllers/VirtualRoomController/GetInterpreterRoomForParticipantTests.cs) in video web

Reply:
Sorry, I'm not changing it to Auto mock, please create a story on the backlog.

Reply:
fair enough now the work is done. next time maybe.